index_V1

```diff
@@ -29,49 +69,141 @@
+				
+				</div>
+				
+				<div class="bar">
+				
+					<form method = "post" style="margin-top: 1vw;">
+						
+					  <input type="text" autocomplete="off" class="search" placeholder="Search..." name="searched">
+						
+					  <button class="button" type="submit" name="search" style="height: 3vw; width: 6vw; background-color: grey; float: right; margin-top: -2.9vw;"><img src="images/icon_6.png" style="width: 1.2vw; height: 1.2vw;"></button>
+						
+					</form>
+				
+				</div>
+			
+			</div>
 			
 			<div class="circles">
 				
 				<div class="table">
 					
-					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 1; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 1; background-color: #e6e6e6; overflow: hidden; border-radius: 1.5vw; margin: 0vw 1.5vw; transition: 0.6s;">
+						
+						<img src="images/img_8.jpg" img draggable="false" alt="Blue" style="width: 32vw; height: 16vw;">
 					
-					<div onclick="location.href='query_1.php';"class="circle_2" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					</div>
 					
-					<div onclick="location.href='query_1.php';"class="circle_3" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					<div onclick="location.href='query_2.php';"class="circle_2" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; overflow: hidden; border-radius: 1.5vw; margin: 0vw 1.5vw; transition: 0.6s;">
+						
+						<img src="images/img_9.jpg" img draggable="false" style="width: 32vw; height: 16vw;" alt="Blue">
+					
+					</div>
 					
-					<div onclick="location.href='query_1.php';"class="circle_4" style="cursor: pointer; grid-column: 4; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
-				
 				</div>
 			
 			</div>
 			
-			<div class="divider"></div>
+			<div class="divider" id="divider" style="background-color: lightgrey; border-radius: 1vw;"></div>
 			
 			<div class="album_1">
 			
 			<div class="circles">
 				
 				<div class="table_2">
 					
-					<div class="text" style="grid-column: 1; background-color: grey; margin: 0vw 1vw;"></div>
+					<div class="text" style="grid-column: 1; background-color: white; margin: 0vw 1vw;">
+					
+						<h3 style="font-size: 2.2vw; font-family: 'Noto Sans JP', sans-serif;">GENRES</h3>
+						
+						<p>Genre Shortcuts we think you will be interested in!</p>
+					
+					</div>
+					
```

Comment:
This is the code that I have added for the Genre Song Albums, including the header and div sizing